### ISSUE TITLE
Error when not using zip feature

### ISSUE THREAD

**Issue by vilaureu** [georust/kml#21]
# Problem
The _kml_ crate does not compile without the _zip_ feature.
```
error[E0433]: failed to resolve: use of undeclared crate or module `zip`
  --> kml/src/errors.rs:34:22
   |
34 |     ZipError(#[from] zip::result::ZipError),
   |                      ^^^ use of undeclared crate or module `zip`
```
# Steps to Reproduce
Create a new crate with cargo and add `kml = { version = "0.4.1", default-features = false }` to the dependencies.
`cargo check` now fails.
# Possible Solution
Adding the line `#[cfg(feature = "zip")]` before `#[error("ZIP error: {0}")]` in `errors.rs` fixes the issue for me.

**Comment by pjsier** [comment]
@vilaureu thanks for reporting this! Do you want to put in a PR with that change? Otherwise I can patch that quick. It probably makes sense to update CI to test different feature combinations as well

**Comment by pjsier** [comment]
Thanks again @vilaureu! This is included in the v0.4.2 release
